src/components/banner/banner.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../app/globals.scss';
```

Comment:
Everything looks great!
Just a small suggestion: consider using `@use` instead of `@import`. Sass is planning to remove `@import`; please refer to [Sass documentation](https://sass-lang.com/documentation/at-rules/import/) for more details.

Reply:
`@use '../../app/globals.scss' as *;`